### PR TITLE
initial commit checking for duplicate sample names

### PR DESCRIPTION
**Pull request by lynnjo**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Verify sample names do not already exist  when adding new VCFs to datasets
```

### CODE REVIEW

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt [hunk 1]
```diff
@@ -63,14 +66,43 @@ class LoadVcf : CliktCommand(help="load g.vcf and h.vcf files into tiledb datase
         if (fileLists.first.isNotEmpty()) {
             val gvcfExists = verifyURI(dbPath,"gvcf_dataset")
             if (gvcfExists){
+                // look for duplicate sample names
+                // Get samples from tiledb
+                val uri = dbPath + "/gvcf_dataset"
```
Comment (zrm22) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:71]: Is this a new file which gets created/filled in?

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt [hunk 2]
```diff
@@ -63,14 +66,43 @@ class LoadVcf : CliktCommand(help="load g.vcf and h.vcf files into tiledb datase
...
                 val datasetURI = "${dbPath}/gvcf_dataset"
                 loadVCFToTiledb(fileLists.first, datasetURI, threads)
             }
         }
         if (fileLists.second.isNotEmpty()) {
             val hvcfExists = verifyURI(dbPath,"hvcf_dataset")
             if (hvcfExists){
+                // look for duplicate sample names
+                // Get samples from tiledb
+                val uri = dbPath + "/gvcf_dataset"
```
Comment (zrm22) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:94]: Update this to hvcf_dataset

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt (hunk 1 above)
Comment (zrm22) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:71]: Nevermind, I understand based on the next bit of code.

**Review by zrm22 — APPROVED** [review]

**Comment by zrm22** [comment]
Also looks like your changes are breaking the Integration Test.

File: src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt (hunk 2 above)
Comment (lynnjo) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/LoadVcf.kt:94]: updated!

**Review by tcasstevens — APPROVED** [review]

**Review by pjbradbury — APPROVED** [review]

**Pull request merged**